i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-os.mdx

```diff
@@ -22,6 +22,25 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
 
 ## Centreon Web
 
+### 24.04.5
+
+Release date: `July 18, 2024`
+
+<details open>
+  <summary>Enhancements</summary>
+
+- [API] Fixed access control for hosts configuration endpoints.
+- [centreon-web] Fixed selinux dependencies for centreon-common-selinux
+- [Dashboard] Empty legend in Metrics Graph widget when user locale is different from en_US
+- [Dashboards] Displayed statuses of parent hosts when only services are selected in status chart and status grid widgets.
```

Comment:
```suggestion
status charts and the status grid widget now display the status of a service's parent host when only services are selected instead of a "no host found" error.
```